### DISCUSSION TITLE
Flags icons for US is wrong (Liberia?)

### DISCUSSION THREAD

**Discussion by kawoolutions** [primefaces/community#1534] [PrimeReact] [answered]
Hey guys.

There's a big mistake in any of the Prime icons' flag icons, more specifically the US flag.

![2024-03-27 21_59_05-](https://github.com/primefaces/community/assets/5111733/7c9aed91-686c-4200-878f-b75e7b3ae434)

See here:

https://github.com/primefaces/primereact/blob/master/public/images/flag/flags_responsive.png

Scroll down the PNG and see two flags below the flag for UK:

![2024-03-27 22_04_29-primereact_public_images_flag_flags_responsive png at master · primefaces_primer](https://github.com/primefaces/community/assets/5111733/a7d42e9d-c136-4d79-afc2-e2b3229379f9)

That's more like the flag of Liberia, but even that seems wrong. Embarassing.

Please fix??

**Comment by melloware** [comment]
Those are from PrimeTek that is just flags.css and PNG they got from somewhere. Probably just need an updated version from wherever PrimeTek got it from.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue, we'll address this issue in the 17.13.0 release.
Issue link: https://github.com/primefaces/primeng/issues/15184

**Comment by kawoolutions** [comment]
I switched to https://github.com/lipis/flag-icons/ entirely. The icon set here is quite old and pixelated anyways. The other one uses SVGs.

Maybe PrimeTek wants to recommend and integrate this into their website going forward.

**Reply by melloware** [reply]
I also found this for those that want 1 PNG Sprite and not SVG :https://www.freakflagsprite.com/
